ANSWERS.md

```diff
@@ -0,0 +1,12 @@
+1. What is the DOM? 
+
+The DOM is an interface that allows a programming language to manipulate and dynamically access and update the content, structure, and style of a website. 
+
```

Comment:
Right.  The DOM is basically a tree of child nodes nested inside of their respective parent nodes.